guides/v2.1/cloud/project/project-conf-files_magento-app.md

```diff
@@ -349,3 +383,39 @@ For a list of recommended PHP configuration settings, see [Required PHP settings
 After pushing your file, you can check that the custom PHP configuration has been added to your environment by [creating an SSH tunnel]({{ page.baseurl }}/cloud/env/environments-start.html#env-start-tunn) to your environment and entering:
 
 	cat /etc/php5/fpm/php.ini
+
+## Workers
```

Comment:
Is this supposed to be here? Originally it was not in 2.1, so just wondering.

Reply:
Removed it.